### DISCUSSION TITLE
Vehicle model not recognized

### DISCUSSION THREAD

**Discussion by kalagan55** [flobz/psa_car_controller#400] [General]
Hello, I have the following error message. can you help me ?
"WARNING :: Can't get car model, please report an issue on github with your car model and first ten letter of your VIN : VF7SXYHYPK"
Thanks

**Comment by rimrul** [comment]
Well, like the error message says, please tell us the car model.

**Comment by kalagan55** [comment]
Bonjour, j'ai une E208 et une C aircross Hybride
